dev-env/bgp/frr/bgpd.conf

```diff
@@ -0,0 +1,39 @@
+! -*- bgp -*-
+!
+! BGPd sample configuration file
+!
+! $Id: bgpd.conf.sample,v 1.1 2002/12/13 20:15:29 paul Exp $
+!
+hostname bgpd
+password zebra
+!enable password please-set-at-here
+!
+router bgp 64512
+ neighbor 172.18.0.2 remote-as 64512
+ neighbor 172.18.0.2 next-hop-self
+ neighbor 172.18.0.3 remote-as 64512
+ neighbor 172.18.0.3 next-hop-self
+ neighbor 172.18.0.4 remote-as 64512
+ neighbor 172.18.0.4 next-hop-self
```

Comment:
bear in mind that the ips provided to the kind nodes are assigned by docker, you are not guarantee to have the same ips or in the same order

Reply:
yeah ... I thought about making this smarter, but it has been the same every time I've run it, so then I got lazy :)

Reply:
if you use for automated testing is risky, if is an example, maybe adding a comment explaining how to obtain those ips?

Reply:
Yeah, good point.  I think it should be automated before relying on it for automated testing.

Reply:
I'll just do it now.

Reply:
OK, this is done now